Supertext.Base.Dal.SqlServer.Tests/Utils/SqlResultConverterTest.cs

```diff
@@ -165,4 +165,180 @@ public void DecodeStructure_JsonEncodedFieldNames_Level2Json()
         (b.GetProperty("c1").GetInt32()).Should().Be(1);
         (b.GetProperty("c2").GetInt32()).Should().Be(2);
     }
+
+    [TestMethod]
+    public void InterpretUtcDates_ObjectWithNonNullableDateTime_ConvertedInUtc()
+    {
+        var testData = new TestEntity(Guid.NewGuid(),
+                                      "Test",
+                                      DateTime.Now,

```

Comment:
I think I would like to see an assertion that `DateTime.Now.Kind` is not already `DateTimeKind.Utc` - this way, you are proving that your method has made the change.